db/schema.rb

```diff
@@ -817,6 +819,8 @@
 
   add_foreign_key "account_facility_joins", "accounts"
   add_foreign_key "account_facility_joins", "facilities"
+  add_foreign_key "account_users", "accounts", name: "fk_accounts"
```

Comment:
These don't seem to have been added by your migration? I'm fine with them being there, just wondering where they came from.

Reply:
🤦‍♂️ You are right! `account_users` to `users came in https://github.com/tablexi/nucore-open/blob/4095962f73d481e67307f55f396cd6d287fb880a/db/migrate/20170508150135_add_foreign_keys.rb#L12-L13 and I believe that `fk_accounts` came as a result of https://github.com/tablexi/nucore-open/blob/4095962f73d481e67307f55f396cd6d287fb880a/db/migrate/20100528212342_add_account_users_table.rb#L7-L8 creating foreign keys by default in Rails, but the schema never got updated along the way.

Reply:
I will take them out of here, though, and fix `schema.rb` separately. Good catch!